<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMeasureTool</class>
 <widget class="QWidget" name="CMeasureTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>112</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Staff</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Distance:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="leDistance"/>
     </item>
     <item>
      <widget class="QComboBox" name="cbUnit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cbPath">
     <property name="text">
      <string>Path</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cbGreatCircle">
     <property name="text">
      <string>Great Circle</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
